### ISSUE TITLE
Change the from in New Post

### ISSUE THREAD

**Issue by mrhanson3** [async-labs/saas#133]
I can't seem to find where to change the name in the From when there's a new post. 

It comes from my email address, but the name is From async-await.com. Any guidance would be appreciated. 

<p/><hr/><details>
      <summary>Click to see <b>Hill for issue #133</b></summary>
      <div><img src="https://async-github-hills.s3.amazonaws.com/async-labs-saas/133.png" alt="Single issue hill" class="s3-image" width="100%" /></div></details>
      <details><summary>Click to see <b>Hill for all issues </b></summary><div>
      <img src="https://async-github-hills.s3.amazonaws.com/async-labs-saas/all-issue-hill.png" alt="All issue hill" class="s3-image" width="100%" /></div></details>
      
created by [Async](https://async-await.com/)

**Comment by mrhanson3** [comment]
That's the only one. Not any others.

**Comment by mrhanson3** [comment]
Further, when changes are made in EmailTemplate.ts, the changes aren't reflected in MongoDb.

**Comment by klyburke** [comment]
@mrhanson3 Thanks for reporting. Looks like you have two issues - the wrong "From" wording for your email about new posts and not seeing updates in `EmailTemplate.ts`

(1) Did you update `from:` in the `handler.ts` file? See here: https://github.com/async-labs/saas/blob/a22ddbb5c642869810ab3571c188edb7357d851a/lambda/handler.ts#L58

(2) To see changes in `EmailTemplate.ts`, you will need to restart your API server. We described it in this section of Chapter 6 of our SaaS Boilerplate book: https://builderbook.org/books/saas-boilerplate/aws-ses-api-passwordless-oauth-api-mailchimp-api#emailtemplate-model-inserttemplates-and-getemailtemplate-methods

As a quick reminder, check under the line that says "Here are some bonus tests you can run by yourself":

> - On your MongoDB Atlas dashboard, you can manually delete either the `EmailTemplate` document or `test.emailtemplates` collection. Then you can restart your API server and see that API recreates both the document and collection.
> - On your MongoDB Atlas dashboard, you can manually edit `Thanks for signing up` to `Thank you for signing up` inside the message parameter. Then restart your API server and see that API updates the value of message to match the hardcoded value. The message will be edited back to `Thanks for signing up`.

**Comment by tima101** [comment]
@mrhanson3 About (2). Please look at the code which is ultimate source of truth:

https://github.com/async-labs/saas/blob/master/api/server/models/EmailTemplate.ts

- `EmailTemplate.updateOne` runs only when `insertTemplates` runs
- `insertTemplates` runs only when `getEmailTemplate` runs
- `getEmailTemplate` runs whenever `API` server sends email, for example, here: https://github.com/async-labs/saas/blob/a22ddbb5c642869810ab3571c188edb7357d851a/lambda/handler.ts#L36

In other words, you can update hardcoded `subject` and/or `message` and changes in database should happen after calling  `getEmailTemplate` method.

@klyburke Restarting server would have worked if we called, say, `insertTemplates` method inside `api/server/server.ts` but we don't. Let me fix this content.

**Comment by mrhanson3** [comment]
1) I took care of changing the wording, and it still says "From async-await.com" in the from section of the email. "From async-await.com" doesn't exist in my code any where, based on my search of the main folder but it's still being pulled, and I don't know from where. 
2) Invitations and everything else are fine. 
I deleted my MongoDB emailtemplates and that didn't work.

**Comment by tima101** [comment]
@mrhanson3 So looks like you are talking about deployed project, not local project. The problem must be with how you deploy your `API` server or how you deploy your `Lambda` function. Both needs to be redeployed if you made changes in them. Also, you need to update value for `EMAIL_SUPPORT_FROM_ADDRESS` of deployed `Lambda` function, on AWS Lambda's dashboard.

You can always add `logger.info(options.from);` before return statement here: https://github.com/async-labs/saas/blob/master/api/server/aws-ses.ts

Then deploy, then initiate sending email. And then look at your Heroku logs for `API` server. If you don't see anything printed - problem with how you deploy either `API` or `Lambda`. If you see wrong value, problem with how you updated value for `EMAIL_SUPPORT_FROM_ADDRESS` of deployed `Lambda` function.

**Comment by mrhanson3** [comment]
It isn't in the Lambda function EMAIL_SUPPORT_FROM_ADDRESS. That was changed already. I don't really need help at this point. I'm just working out the issue in case anyone down the road has the same problem. Let me dig in to redeployment.

**Comment by tima101** [comment]
@mrhanson3 It is in original code: https://github.com/async-labs/saas/blob/a22ddbb5c642869810ab3571c188edb7357d851a/lambda/handler.ts#L58

If you have codebase that differs from this repo, I can't help.

**Comment by mrhanson3** [comment]
It isn't different. It's the codebase you set up. The only change is I removed the From async-await.com 
<img width="377" alt="Screen Shot 2020-10-20 at 2 21 54 PM" src="https://user-images.githubusercontent.com/20564536/96628260-fc9bad80-12df-11eb-9b71-9b4cdc7cb32d.png">

This is my code: 
![Screen Shot 2020-10-20 at 2 25 57 PM](https://user-images.githubusercontent.com/20564536/96628442-3a003b00-12e0-11eb-8b0d-fd2606eee125.png)

It isn't changing no matter what I do.

**Comment by tima101** [comment]
@mrhanson3 Like I said in the previous comment, you have to print values using `console.log` on our deployed `API` server and deployed Lambda function. This will should give a definitive answer. It's most like that deployed code is not updated, in other words, something wrong with deploying process.

There is no magic here really, it's logic. if you removed some string from code and database, and successfully deployed projects, and updated env variables - you should see this change on your transactional emails.

**Comment by mrhanson3** [comment]
The only one that doesn't update is the handler.ts. Doesn't matter. I've changed so that it isn't an option when creating discussions. Given that you set it up, it's a little disappointing, but I've moved on from it. I blame myself for not having more rigorous testing before the contract closed. But I'm done with this.
